plugins/pulp_rpm/plugins/distributors/yum/metadata/updateinfo.py

```diff
@@ -16,64 +14,45 @@
 
 
 class UpdateinfoXMLFileContext(XmlFileContext):
-    def __init__(self, working_dir, checksum_type=None, conduit=None):
+
+    def __init__(self, working_dir, nevra_in_repo, checksum_type=None, conduit=None):
```

Comment:
A doc block would be helpful, even if it only lists the parameter you just added.